cabal.project

```diff
@@ -6,9 +6,11 @@ allow-newer: hackage-db:Cabal, hpack:Cabal
 
 source-repository-package
     type: git
-    location: https://github.com/hamishmack/hackage-db.git
-    tag: 173783814c9e2d7b80d5d4b5a628334361a0b34a
-    --sha256: sha256-T6cKhYvCaWua6S87PYo6LrOvhg2JGjdjm2XLan9300Y=
+    location: https://github.com/michaelpj/hackage-db.git
+    tag: f3b9240212b036391871e4ea09891e91efcea7a1
+    --sha256: sha256-n0ATmkwtR68E2FuZK3QIQgZirVmWbd21vIQmzhGKsRw=
+
+constraints: Cabal >= 3.4
```

Comment:
I don't think you need this here, if anywhere it should be in the cabal file, since we use `Distribution.Utils.Path` which isn't introduced until 3.4.